benchmarks/transform_warp.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from skimage.transform import SimilarityTransform, warp
+from skimage.util.dtype import convert
+
+
+class WarpSuite:
+    params = ([np.uint8, np.uint16, np.float32, np.float64],
+              [128, 1024, 4096],
+              [0, 1, 3]
+              )
+    param_names = ['dtype_in', 'N', 'order']
+
+    def setup(self, dtype_in, N, order):
+        self.image = convert(np.random.rand(N, N), dtype=dtype_in)
+        self.tform = SimilarityTransform(scale=1, rotation=np.pi / 10,
+                                         translation=(0, 4))
+        self.order = order
+
+    def same_type(self):
+        """Test the case where the users wants to preserve their same low
+        precision data type."""
+        result = warp(self.image, self.tform, order=self.order,
+                      preserve_range=True)
+        convert(result, dtype=self.image.dtype, copy=False)
```

Comment:
Right now you are measuring the time it takes to execute `warp` *and* `convert` together. Is this intended? Wouldn't it be better to keep these measurements separate?